### ISSUE TITLE
TinyMCE not working with elemental if loaded in react context after already being loaded (silverstripe/admin 4.13)

### ISSUE THREAD

**Issue by baukezwaan** [silverstripe/silverstripe-admin#1512]
Ok, this is kind of an edge case. But really annoing if it is your specific setup, because clients get really weird experiences. 

## Step to reproduce (from editor perspective):
This is when you open the page (not when ajax-navigating, only on first pageload):
![image](https://github.com/silverstripe/silverstripe-elemental/assets/795238/71c710ea-b938-44df-bdae-b45ef7065301)
This is when you open a elemental block:
![image](https://github.com/silverstripe/silverstripe-elemental/assets/795238/eebcb58f-8968-4e99-ac82-ea48b0bd994b)

## Step to reproduce:
1. Install clean silverstripe via composer:

  ```
  "silverstripe/recipe-cms": "4.13.*",
  "dnadesign/silverstripe-elemental": "*"
  ```

2. Setup Page for Elemental use via yml:

```
Page:
  extensions:
    - DNADesign\Elemental\Extensions\ElementalPageExtension
```

3. Setup Page.php for use with HTMLEditor field from Page

```private static $elemental_keep_content_field = true;```

4. Open the CMS and add one or two standard Content-elemental field to the page and save the page.

5. Refresh the page  and open the first Element, and see weird stuff happening. Again, the part 'refresh the page' is important here. The issue does not seem to occur on navigating after pageload / via AJAX.

## Possible solutions / found problems
- The ID of all the tinyMCE fields is the same, when inspectie the source. This explains why the text-part of the editor is made visible in the wrong place
- This seems to be introduced in the 4.13 release of the silverstripe/admin
- Probably somewhere in the update from tinymce from @GuySartorelli (https://github.com/silverstripe/silverstripe-admin/commit/368575b4a994932615512c4537da09e98f1d84cf)

So the quick-fix for production environments:
- Don't upgrade tot 4.13 yet:

```
        "silverstripe/recipe-cms": "4.12.*",
        "dnadesign/silverstripe-elemental": "*"
```

- Update to 4.13, but don't use silverstripe/admin 4.13 and stick to 4.12

```
        "php": "^7.4 || ^8.0",
        "silverstripe/recipe-plugin": "^1.7",
        "silverstripe/recipe-core": "~4.12.0@stable",               <-----------
        "silverstripe/admin": "~1.13.0@stable",
        "silverstripe/asset-admin": "~1.13.0@stable",
        "silverstripe/campaign-admin": "~1.13.0@stable",
        "silverstripe/versioned-admin": "~1.13.0@stable",
        "silverstripe/cms": "~4.13.0@stable",
        "silverstripe/errorpage": "~1.13.0@stable",
        "silverstripe/reports": "~4.13.0@stable",
        "silverstripe/siteconfig": "~4.13.0@stable",
        "silverstripe/versioned": "~1.13.0@stable",
        "silverstripe/graphql": "~3.8.1@stable || ~4.3.0@stable",
        "silverstripe/session-manager": "~1.5.0@stable"

        "dnadesign/silverstripe-elemental": "*"
```

- Update to Silversripe 5 (the problem does not exsist there either)

```
        "silverstripe/recipe-cms": "^5",
        "dnadesign/silverstripe-elemental": "^5"
```

## Acceptance criteria
- Multiple HTML Editor can be edited at the same time on the same page. This includes:
  - 2 HTML editors out of elemental
  - 1 HTML editor in elemental and 1 out elemental
  - 2 HTML editors in separate Elemental blocks

## PRs
- https://github.com/silverstripe/silverstripe-admin/pull/1525

**Comment by GuySartorelli** [comment]
From what you've described, it sounds like this is a problem in `silverstripe/admin` and would be reproducible in some way by having multiple WYSIWYG fields in a form. I'm going to move it over to that repository.

**Comment by baukezwaan** [comment]
True, it is more related to silverstripe-admin because of the TinyMCE installation.

But I just tested it without Elemental:
- multiple HTMLFields is working without problems
- It seems to be related to the HTMLField before the elemental-area in the CMS

One new finding:
- Adding an extra HTMLField to Page, moves the problem to the _second_ Elemental-element

**Comment by GuySartorelli** [comment]
Awesome, thanks for that context.

**Comment by GuySartorelli** [comment]
Note for anyone trying to reproduce this in the future: Sometimes you need to open and close the elemental block several times before it'll trigger the behaviour described above.

**Comment by LABCAT** [comment]
I have also experienced this issue today but I have elemental_keep_content_field set to false.

I did have other HTML fields (eg Intro, AltIntro) so I think the issue is more related to elemental rendering a new TinyMCE when there are already existing TinyMCE fields showing outside of elemental.

As a simple workaround I can move the fields to a different tab.

**Comment by dizzystuff** [comment]
I just logged an issue before I read this one 🤦‍♂️ , I believe they're similar symptoms of the same issue - https://github.com/silverstripe/silverstripe-admin/issues/1519

**Comment by LABCAT** [comment]
I have done this in my project which resolved both this issue and the one described in #1519 :

`"silverstripe/admin": "1.12.7 as 1.13.1",`

Not sure what the repercussions will be though.

**Comment by GuySartorelli** [comment]
It's not clear why this only started being a problem after the TinyMCE update - but this was ultimately caused by loading the tinymce script twice. PR incoming.

**Comment by emteknetnz** [comment]
Linked PR has been merged and released as [1.13.4](https://github.com/silverstripe/silverstripe-admin/releases/tag/1.13.4) and [2.0.3](https://github.com/silverstripe/silverstripe-admin/releases/tag/2.0.3)
